lib/node_modules/@stdlib/stats/base/dists/betaprime/mode/src/main.c

```diff
@@ -0,0 +1,69 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/betaprime/mode.h"
+
+/**
+* Returns the mode of a beta prime distribution with first shape parameter `alpha` and second shape parameter `beta`.
+*
+* @param x        input value
+* @param alpha    first shape parameter
+* @param beta     second shape parameter
+* @returns        evaluated logPDF
+*
+* @example
+* double y = stdlib_base_dists_betaprime_mode( 1.0, 2.0 );
+* // returns ~0.0
+*
+* @example
+* double y = stdlib_base_dists_betaprime_mode( 4.0, 12.0 );
+* // returns ~0.231
+*
+* @example
+* double y = stdlib_base_dists_betaprime_mode( 8.0, 2.0 );
+* // returns ~2.333
+*
+* @example
+* double y = stdlib_base_dists_betaprime_mode( 1.0, 1.0 );
+* // returns ~0.0
+*
+* @example
+* double y = stdlib_base_dists_betaprime_mode( 1.0, -0.1 );
+* // returns NaN
+*
+* @example
+* double y = stdlib_base_dists_betaprime_mode( -0.1, 1.0 );
+* // returns NaN
+*
+* @example
+* double y = stdlib_base_dists_betaprime_mode( 2.0, NaN );
+* // returns NaN
+*
+* @example
+* double y = stdlib_base_dists_betaprime_mode( NaN, 2.0 );
+* // returns NaN
```

Comment:
Let's just keep one example in the C source files per convention:

```suggestion
* @example
* double y = stdlib_base_dists_betaprime_mode( 1.0, 2.0 );
* // returns ~0.0
```